### ISSUE TITLE
Week 41 - magneto

### ISSUE THREAD

**Issue by jonallured** [jonallured/dotfiles#155]
## Terminal Stuff

* [x] `$ brew upgrade bat fzf git heroku hokusai hub macvim the_silver_searcher tmux yarn`
* [x] `$ replug`
* [x] `$ asdf update && asdf plugin-update --all && asdf reshim`
* [x] `$ gem update --system`
* [x] `$ npm install -g npm`
* [x] `$ nuke_modules`

## Filesystem Housekeeping

* [x] cleanup desktop
* [x] cleanup screenshots
* [x] cleanup downloads
* [x] empty trash

## Check for Updates

* [x] Alfred
* [x] iTerm2
* [x] 1Password
* [x] Sketch
* [x] Zeplin
* [x] do App Store.app updates
* [ ] do System Updates

## Stay in Sync

* [x] push/pull changes on dotfiles
* [x] push/pull changes on notes
* [x] push/pull changes on secrets
* [x] update list of Artsy repos

## Finishing Up

* [x] check free disk space
* [x] verify last backup on time machine
* [x] reboot

**Comment by jonallured** [comment]
## sharpen.log

<details>

```
+ brew upgrade bat fzf git heroku hokusai hub macvim the_silver_searcher tmux yarn
Updating Homebrew...
==> Auto-updated Homebrew!
Updated Homebrew from 6c2b9ffdc to aa043c1d6.
Updated 4 taps (heroku/brew, homebrew/core, homebrew/cask and homebrew/services).
==> New Formulae
alp
appium
arduino-cli
atasm
dafny
diffr
gleam
kyma-cli
libtensorflow@1
mysql-connector-c++@1.1
notifiers
numpy@1.16
pnetcdf
wagyu
xgboost
==> Updated Formulae
aliyun-cli
amazon-ecs-cli
amtk
angle-grinder
angular-cli
anjuta
apache-arrow
apache-arrow-glib
apache-geode
apache-zeppelin
aria2
armadillo
ask-cli
augustus
aws-cdk
aws-okta
aws-sdk-cpp
awscli
b2-tools
balena-cli
ballerina
baobab
bear
bee
bit
bnd
broot
buildkit
caffe
cfn-lint
cgal
clib
cloc
clutter-gtk
cmake
cnats
cocoapods
collectd
conan
conjure-up
convox
cromwell
crystal
deno
devspace
dhall
diff-pdf
django-completion
doctl
dos2unix
dosbox-x
drone-cli
embulk
eslint
ethereum
ettercap
evince
expat
exploitdb
faas-cli
fetchmail
ffsend
file-roller
firebase-cli
flow
fluid-synth
fmt
fn
folly
fonttools
fribidi
frugal
gammu
gdl
gedit
geos
ghex
gitg
gjs
glade
glib
glib-networking
glooctl
gmime
gnome-builder
gnome-latex
gnome-recipes
gnumeric
gnuradio
gnutls
go@1.12
gobby
goffice
goreleaser
grafana
graphviz
groonga
grpc
gsmartcontrol
gspell
gtk+3
gtk-mac-integration
gtkmm3
gtksourceview3
gtksourceview4
gtksourceviewmm3
gtkspell3
gtranslator
gupnp-tools
haproxy
harfbuzz
helmfile
heroku/brew/heroku
homebank
hub
icdiff
igv
imagemagick
iso-codes
istioctl
jenkins
jenkins-lts
joplin
juju
k3d
kcov
kerl
klavaro
kops
kubeaudit
kubebuilder
kubernetes-cli
kubeseal
kustomize
lerna
libcaca
libdazzle
libfabric
libgweather
libpeas
libphonenumber
libpulsar
librsvg
libtensorflow
libvirt
lightning
lmod
luarocks
lxc
minio
minio-mc
minizinc
minizip2
mkl-dnn
mosh
nano
ne
netlify-cli
newman
newsboat
nnn
node
now-cli
nss
numpy
nvm
ola
opa
openapi-generator
osm-gps-map
p11-kit
packer
paket
parallel
pdftoipe
pgweb
phoronix-test-suite
php@7.2
phpmyadmin
phpunit
picard-tools
poppler
prefixsuffix
prometheus
protobuf
protobuf-c
pspg
pulumi
qalculate-gtk
quickjs
rancher-cli
rclone
redis
rke
roswell
ruby
ruby-build
rust
salt
sccache
scipy
screen
serf
shairport-sync
shared-mime-info
ship
shogun
simple-scan
sip
siril
skafos
sonobuoy
sops
sourcekitten
spice-gtk
sqlmap
starship
step
syncthing
telegraf
teleport
tepl
terraform
terraformer
terragrunt
tmuxinator
tmuxinator-completion
topgrade
traefik@1
trafficserver
unbound
v2ray-plugin
v8
vala
varnish
vert.x
vte3
vulkan-headers
wsk
wtf
xonsh
xrootd
xsimd
yaml-cpp
yash
zbackup
zint
znc
zsh-completions
==> Deleted Formulae
pbrt

Error: bat 0.12.1 already installed
Error: fzf 0.18.0 already installed
Error: git 2.23.0_1 already installed
Error: artsy/formulas/hokusai 0.5.7 already installed
Error: macvim 8.1-157 already installed
Error: the_silver_searcher 2.2.0 already installed
Error: tmux 2.9a_1 already installed
Error: yarn 1.19.0 already installed
==> Upgrading 2 outdated packages:
heroku/brew/heroku 7.31.2 -> 7.33.1
hub 2.12.7 -> 2.12.8
==> Upgrading heroku/brew/heroku 
==> Downloading https://cli-assets.heroku.com/heroku-v7.33.1/heroku-v7.33.1.tar.xz
==> Caveats
To use the Heroku CLI's autocomplete --
  Via homebrew's shell completion:
    1) Follow homebrew's install instructions https://docs.brew.sh/Shell-Completion
        NOTE: For zsh, as the instructions mention, be sure compinit is autoloaded
              and called, either explicitly or via a framework like oh-my-zsh.
    2) Then run
      $ heroku autocomplete --refresh-cache

  OR

  Use our standalone setup:
    1) Run and follow the install steps:
      $ heroku autocomplete

Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/heroku/7.33.1: 18,576 files, 49.1MB, built in 21 seconds
==> `brew cleanup` has not been run in 30 days, running now...
Removing: /usr/local/Cellar/heroku/7.31.2... (18,716 files, 49.5MB)
Removing: /Users/jon/Library/Caches/Homebrew/heroku--7.31.2.tar.xz... (6.3MB)
Removing: /Users/jon/Library/Caches/Homebrew/heroku-node--11.14.0.tar.xz... (8.3MB)
Removing: /Users/jon/Library/Caches/Homebrew/hub--2.12.7.mojave.bottle.tar.gz... (4.6MB)
Removing: /Users/jon/Library/Caches/Homebrew/icu4c--64.2.mojave.bottle.tar.gz... (26.1MB)
Removing: /Users/jon/Library/Caches/Homebrew/jrnl--1.9.8_1.mojave.bottle.tar.gz... (3.0MB)
Removing: /Users/jon/Library/Caches/Homebrew/node--12.10.0.mojave.bottle.tar.gz... (14.7MB)
Removing: /Users/jon/Library/Caches/Homebrew/openssl--1.0.2s.mojave.bottle.tar.gz... (3.7MB)
Removing: /Users/jon/Library/Caches/Homebrew/readline--8.0.0_1.mojave.bottle.tar.gz... (517.7KB)
Removing: /Users/jon/Library/Logs/Homebrew/python... (3 files, 131.9KB)
Removing: /Users/jon/Library/Logs/Homebrew/sqlite... (64B)
Removing: /Users/jon/Library/Logs/Homebrew/memcached... (64B)
Removing: /Users/jon/Library/Logs/Homebrew/macvim... (64B)
Removing: /Users/jon/Library/Logs/Homebrew/tmux... (64B)
Removing: /Users/jon/Library/Logs/Homebrew/libevent... (64B)
Removing: /Users/jon/Library/Logs/Homebrew/bat... (64B)
Removing: /Users/jon/Library/Logs/Homebrew/hub... (64B)
Removing: /Users/jon/Library/Logs/Homebrew/git... (64B)
Pruned 0 symbolic links and 1 directories from /usr/local
==> Upgrading hub 
==> Downloading https://homebrew.bintray.com/bottles/hub-2.12.8.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/9a/9ac84d4f756253267655ca78c1d4eed783f63d3810e1d796e913a7c1a11cafcd?__gda__=exp=1570562115~hmac=8b45d1d01c01b794de2cd17b942506e38d96111de1a08980e1e20bf6a0a76ccd&response-content-disposition=attachment%3Bfilename%3D%22hub-2.12.8.mojave.bottle.tar.gz%22&response-content-type=application%2Fgzip&requestInfo=U2FsdGVkX189-AWO05d7YCeKI7m9gGViVAEA_Jia_bn49fRnFX7LfAGEZ3GETL3R-0nAGo9iAQazldLJrtKcLC_cJxlpq1NJvMrtZYDDNgfakk3Z6-BuJeDLTnz5z-rNkEvNvruy0WweOiEKhY9-xA&response-X-Checksum-Sha1=51131d69d16fae8882814203f290a78fe00796c0&response-X-Checksum-Sha2=9ac84d4f756253267655ca78c1d4eed783f63d3810e1d796e913a7c1a11cafcd
==> Pouring hub-2.12.8.mojave.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/hub/2.12.8: 36 files, 9.0MB
Removing: /usr/local/Cellar/hub/2.12.7... (36 files, 9.0MB)
==> Caveats
==> heroku
To use the Heroku CLI's autocomplete --
  Via homebrew's shell completion:
    1) Follow homebrew's install instructions https://docs.brew.sh/Shell-Completion
        NOTE: For zsh, as the instructions mention, be sure compinit is autoloaded
              and called, either explicitly or via a framework like oh-my-zsh.
    2) Then run
      $ heroku autocomplete --refresh-cache

  OR

  Use our standalone setup:
    1) Run and follow the install steps:
      $ heroku autocomplete

Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> hub
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
+ set -e
+ asdf update
HEAD is now at 58eaad8 Update version to 0.7.4
Updated asdf to release v0.7.4
+ asdf plugin-update --all
Updating nodejs...
From https://github.com/asdf-vm/asdf-nodejs
   8bbe1ed..7e43853  master     -> origin/master
Updating 8bbe1ed..7e43853
Fast-forward
 bin/install | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
Current branch master is up to date.
Updating python...
Already up to date.
Current branch master is up to date.
Updating ruby...
From https://github.com/asdf-vm/asdf-ruby
   c17d4d1..0bb80da  master     -> origin/master
Updating c17d4d1..0bb80da
Fast-forward
 lib/utils.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
Current branch master is up to date.
+ asdf reshim
+ gem update --system
Latest version already installed. Done.
+ npm install --global npm
/Users/jon/.asdf/installs/nodejs/10.15.1/.npm/bin/npx -> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/bin/npx-cli.js
/Users/jon/.asdf/installs/nodejs/10.15.1/.npm/bin/npm -> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/bin/npm-cli.js

> minipass@2.8.6 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/fs-minipass/node_modules/minipass
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> fs-minipass@1.2.7 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/fs-minipass
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> hosted-git-info@2.8.5 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/hosted-git-info
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> minizlib@1.2.2 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/minizlib
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> stringify-package@1.0.1 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/stringify-package
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> minipass@2.8.6 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/tar/node_modules/minipass
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> tar@4.4.12 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/tar
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> node-gyp@5.0.5 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/node-gyp
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> npm-lifecycle@3.1.4 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/npm-lifecycle
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> libcipm@4.0.4 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/libcipm
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> npm-install-checks@3.0.2 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm/node_modules/npm-install-checks
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall

> npm@6.12.0 postinstall /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/npm
> /Users/jon/.asdf/installs/nodejs/10.15.1/.npm/lib/node_modules/.hooks/postinstall
+ npm@6.12.0
added 1 package from 1 contributor, removed 1 package and updated 11 packages in 6.651s
+ nuke_modules
/Users/jon/code/palette/node_modules
830M cleaned up

```

</details>

## replug.log

<details>

```
+ PlugClean
Searching for invalid plugins in /Users/jon/.vim/plugs
[=========================================================]

Already clean.
+ PlugInstall
Updated. Elapsed time: 2.240478 sec.
[========================================================]

- Finishing ... Done!
- bufkill.vim: Already installed
- fzf: Already installed
- vim-hashrocket: Already installed
- vim-rake: Already installed
- vim-turbux: Already installed
- vim-pug: Already installed
- vim-ruby: Already installed
- vim-textobj-user: Already installed
- vim-scriptease: Already installed
- vim-json: Already installed
- vim-rsi: Already installed
- tslime.vim: Already installed
- vim-vividchalk: Already installed
- vim-coffee-script: Already installed
- vim-rhubarb: Already installed
- vim-markdown: Already installed
- vim-eunuch: Already installed
- vim-vinegar: Already installed
- vim-repeat: Already installed
- vim-slim: Already installed
- vim-git: Already installed
- vim-commentary: Already installed
- vim-blockle: Already installed
- vim-endwise: Already installed
- supertab: Already installed
- vim-sleuth: Already installed
- ack.vim: Already installed
- vim-spacejam: Already installed
- bufexplorer.zip: Already installed
- vim-railscasts: Already installed
- vim-unimpaired: Already installed
- vim-headfirst: Already installed
- vim-bdubs: Already installed
- vim-rails: Already installed
- vim-weefactor: Already installed
- pgsql.vim: Already installed
- vim-sensible: Already installed
- gitv: Already installed
- vim-irblack: Already installed
- vim-fugitive: Already installed
- vim-textobj-rubyblock: Already installed
- tabular: Already installed
- vim-speeddating: Already installed
- ale: Already installed
- vim-bundler: Already installed
- vim-haml: Already installed
- splitjoin.vim: Already installed
- vim-jsx: Already installed
- vim-abolish: Already installed
- vim-projectionist: Already installed
- vim-ragtag: Already installed
- vim-surround: Already installed
- typescript-vim: Already installed
- vim-javascript: Already installed
- vim-cucumber: Already installed
- vim-styled-components: Already installed
+ PlugUpdate
Updated. Elapsed time: 4.869382 sec.
[========================================================]

- Finishing ... Done!
- bufkill.vim: Already up to date.
- fzf:  1 file changed, 9 insertions(+), 3 deletions(-)
- vim-hashrocket: Already up to date.
- vim-rake: Already up to date.
- vim-turbux: Already up to date.
- vim-pug: Already up to date.
- vim-ruby: Already up to date.
- vim-textobj-user: Already up to date.
- vim-scriptease: Already up to date.
- vim-json: Already up to date.
- vim-rsi: Already up to date.
- tslime.vim: Already up to date.
- vim-vividchalk: Already up to date.
- vim-coffee-script: Already up to date.
- vim-rhubarb: Already up to date.
- vim-markdown: Already up to date.
- vim-eunuch: Already up to date.
- vim-vinegar: Already up to date.
- vim-repeat: Already up to date.
- vim-slim: Already up to date.
- vim-git: Already up to date.
- vim-commentary: Already up to date.
- vim-blockle: Already up to date.
- vim-endwise: Already up to date.
- supertab: Already up to date.
- vim-sleuth: Already up to date.
- ack.vim: Already up to date.
- vim-spacejam: Already up to date.
- bufexplorer.zip: Already up to date.
- vim-railscasts: Already up to date.
- vim-unimpaired: Already up to date.
- vim-headfirst: Already up to date.
- vim-bdubs: Already up to date.
- vim-rails: Already up to date.
- vim-weefactor: Already up to date.
- pgsql.vim: Already up to date.
- vim-sensible: Already up to date.
- gitv: Already up to date.
- vim-irblack: Already up to date.
- vim-fugitive:  2 files changed, 194 insertions(+), 101 deletions(-)
- vim-textobj-rubyblock: Already up to date.
- tabular: Already up to date.
- vim-speeddating: Already up to date.
- ale:  create mode 100644 test/handler/test_debride_handler.vader
- vim-bundler: Already up to date.
- vim-haml: Already up to date.
- splitjoin.vim: Already up to date.
- vim-jsx: Already up to date.
- vim-abolish:  1 file changed, 1 insertion(+), 1 deletion(-)
- vim-projectionist: Already up to date.
- vim-ragtag: Already up to date.
- vim-surround: Already up to date.
- typescript-vim: Already up to date.
- vim-javascript: Already up to date.
- vim-cucumber: Already up to date.
- vim-styled-components: Already up to date.
+ PlugDiff
4 plugin(s) updated.
[========================================================]

Last update:
------------

- ale:
  * f932211 fix tflint handler for 0.11+ (#2775) (24 hours ago)
  * 6d88801 Clarify that virtualtext is only in NeoVim (24 hours ago)
  *   59b34dd Merge pull request #2813 from werneta/master (24 hours ago)
  |\
  | * d0e87c0 Correct vint-discovered advisory in vlog handler (3 days ago)
  | * 89fa435 Add test for new vlog format (3 days ago)
  | * 98b0dcd Update vlog parser to handle new output format (3 days ago)
  * |   f2b231d Merge pull request #2771 from gpanders/master (24 hours ago)
  |\ \
  | * | 7795898 Add tests for clangtidy with Makefile (2 days ago)
  | * | 06a97a8 Use Makefile output with clang-tidy when useful (2 days ago)
  | |/
  * |   8fda1ce Merge pull request #2814 from andys8/improvement/elm-ls-name-fix-in-documentation (25 hours ago)
  |\ \
  | * | 0fa2d18 elm-ls: Removed some leftovers of previous name (3 days ago)
  | |/
  * |   8097d51 Merge pull request #2750 from andys8/improvement/elm-language-server-elm-analyse-trigger-param (25 hours ago)
  |\ \
  | * | bd1f7e1 Elm language server params update (5 weeks ago)
  |  /
  * |   06082ef Merge pull request #2735 from zoonfafer/scala-metals (25 hours ago)
  |\ \
  | * | 5fcb24b Add linter for Scala Metals (6 weeks ago)
  |  /
  * |   d3a3f40 Merge pull request #2676 from davidtwco/nixfmt-fixer (25 hours ago)
  |\ \
  | * | 6aeb462 Add nixpkgs-fmt fixer. (6 weeks ago)
  |  /
  * |   cebbb67 Merge pull request #2659 from greg0ire/psalm-language-server-option (25 hours ago)
  |\ \
  | * | e47c196 Use the psalm executable with the LS option (3 months ago)
  |  /
  * |   32b7842 Merge pull request #2694 from elebow/add-ruby-debride (25 hours ago)
  |\ \
  | |/
  |/|
  | * 8aa1578 Correct copied typo in doc/ale-ruby (8 weeks ago)
  | * 501af8d debride: Add debride linter (closes #2471) (8 weeks ago)
  * a486aa1 Merge pull request #2809 from hsanson/2802-fix-sign-group-parsing (4 days ago)
  * 7c2f5e8 Fix sign parser to be language independent. (4 days ago)

- fzf:
  * 3089880 [vim/windows] Fix chcp parsing for the current codepage (18 hours ago)

- vim-abolish:
  * 3c7f939 Fix invalid sid expression (7 days ago)

- vim-fugitive:
  * 6980655 Fix incorrect script id (part 2) (26 hours ago)
  * 28abadc Fix incorrect script id from nested function call (27 hours ago)
  * a3ba0ca Fix status diffs broken by missing sub attribute (3 days ago)
  * 973d814 Rudimentary support for diff maps for submodules in status (4 days ago)
  * a21ca7e Support X for submodules (4 days ago)
  * f779b90 Add --porcelain=v2 status parser (4 days ago)
  * 80aa8a9 Fix reverse blame jump for the boundary edge case (6 days ago)
  * c9cf1b3 Make remaining buffer commands global (6 days ago)
  * cfdfc1b Clearer error message on attempt to delete directory (6 days ago)
  * 5eeb762 Expand status file lookup into nested structure (7 days ago)
  * 4f0905e Preserve line number on :Gedit with no or > argument (7 days ago)
  * f72ca2e Improve ++opt +cmd handling for :Gedit/:Gread (8 days ago)

+ PlugUpgrade
diff --git a/rcm/vim/autoload/plug.vim b/rcm/vim/autoload/plug.vim
index bc3706c..2571fe3 100644
--- a/rcm/vim/autoload/plug.vim
+++ b/rcm/vim/autoload/plug.vim
@@ -353,9 +353,15 @@ if s:is_win
 
   " Copied from fzf
   function! s:wrap_cmds(cmds)
-    return map(['@echo off','setlocal enabledelayedexpansion','for /f "delims=: tokens=2" %%a in (''chcp'') do set origchcp=%%a','set origchcp=!origchcp: =!','chcp 65001 > nul'] +
-    \ (type(a:cmds) == type([]) ? a:cmds : [a:cmds]) +
-    \ ['chcp !origchcp! > nul','setlocal disabledelayedexpansion'],'v:val."\r"')
+    return map([
+      \ '@echo off',
+      \ 'setlocal enabledelayedexpansion',
+      \ 'for /f "tokens=*" %%a in (''chcp'') do for %%b in (%%a) do set origchcp=%%b',
+      \ 'chcp 65001 > nul'
+    \ ]
+    \ + (type(a:cmds) == type([]) ? a:cmds : [a:cmds])
+    \ + ['chcp !origchcp! > nul', 'endlocal'],
+    \ 'v:val."\r"')
   endfunction
 
   function! s:batchfile(cmd)
+ replug done

```

</details>

**Comment by jonallured** [comment]
Skipping the upgrade to Catalina for now.
